rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -74,9 +74,16 @@ def fixture_data_migration_version() -> int:
     return LAST_DATA_MIGRATION
 
 
+@pytest.fixture(name='load_default_cli_configuration', scope='session')
+def fixture_load_default_cli_configuration() -> int:
```

Comment:
This is not needed. Customize the args. Not have a fixture like this saying use defaults or "some custom you define only for tests".